addon/-private/system/relationships/ext.js

```diff
@@ -12,8 +12,8 @@ var Map = Ember.Map;
 var MapWithDefault = Ember.MapWithDefault;
 
 var relationshipsDescriptor = Ember.computed(function() {
-  if (Ember.testing === true && relationshipsDescriptor._cacheable === true) {
-    relationshipsDescriptor._cacheable = false;
+  if (Ember.testing === true && relationshipsDescriptor._volatile === true) {
```

Comment:
Hum... seems not to be so wonderfull, the tests are now failing... I'm not sure I'm understand, or something else should be changed too ?